cluster/kubernetes/kubernetes.go

```diff
@@ -119,12 +120,15 @@ func NewCluster(client ExtendedClient, applier Applier, sshKeyRing ssh.KeyRing,
 
 // --- cluster.Cluster
 
-// SomeWorkloads returns the workloads named, missing out any that aren't
-// accessible in the cluster. They do not necessarily have to be returned
-// in the order requested.
+// SomeWorkloads returns the workloads named, missing out any that don't
```

Comment:
@2opremio Did I accidentally revert this when rebasing?

Reply:
I am not sure what you mean. Mind clarifying?

Reply:
There was a conflict in this comment when I was rebasing, and I'm not sure I resolved it correctly. It reads OK, I just wanted to make sure I hadn't undone a change you'd carefully made.

Reply:
oh, no, it looks fine :)